front/EditProfileButton.tsx

```diff
@@ -14,7 +14,7 @@ const EditProfileButton = ({ isCurrentUser }: EditProfileButtonProps) => (
       href={routes.userEdit()}
       className="btn btn-sm btn-outline-secondary action-btn"
     >
-      <i className="ion-gear-a" /> Edit Profile Settings
+      <i data-cy='editSettings' className="ion-gear-a" /> Edit Profile Settings
```

Comment:
The 'data-cy' attribute value should follow a consistent naming convention. It is recommended to use kebab-case (lowercase letters with hyphens between words) for HTML attribute values.